packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -453,7 +444,7 @@ export const PersonalDetailsForm = observer(({ history }) => {
                 'is_authenticated_payment_agent',
                 'user_hash',
                 'country',
-                (!is_appstore || !is_eu) && 'salutation',
```

Comment:
is it fine if we're removing `!is_eu` condition?

Reply:
It’s fine, as 'salutation' was used for rendering only if is_appstore and is_eu are true simultaneously

Reply:
I see there is an OR condition and in my understanding salutation is only for eu. Right?

Reply:
the condition for rendering salutation is `is_appstore && is_eu`